netbox/dcim/filtersets.py

```diff
@@ -1219,6 +1219,28 @@ class DeviceComponentFilterSet(django_filters.FilterSet):
         to_field_name='name',
         label=_('Device (name)'),
     )
+    device_type_id = django_filters.ModelMultipleChoiceFilter(
```

Comment:
Could you add filtersets tests for these under the component filterset test cases please?